Makefile

```diff
@@ -40,13 +40,11 @@ help: ## display this help message
 	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
 .PHONY: help
 
-bootstrap: build ## bootstrap the project
-	$(COMPOSE_RUN_LMS) python manage.py lms migrate
+bootstrap: run migrate ## bootstrap the project
```

Comment:
you don't want to `build` before running ? You suppose that tunning the application will build it at the first time ?

Reply:
Yes it will.

Reply:
Ok, for me this is not what I expect from `bootstrap`, but it's ok, keep it like this.